### DISCUSSION TITLE
tap connected network port status

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#10028] [General]
hi,

i have a forward server with 6 ports connected to 6 hardware taps.
how can monitor their status ? how can i know that they send data ?

**Comment by robbiemarshall** [comment]
One option would be to utilize tcpdump to verify traffic is passing to your monitor interfaces.

**Reply by udi-mosh** [reply]
can i automate it and create a graphana graph ?
